src/sampling.jl

```diff
@@ -105,12 +105,12 @@ Draw a pair of distinct integers between 1 and `n` without replacement.
 Optionally specify a random number generator `rng` as the first argument
 (defaults to `Random.GLOBAL_RNG`).
 """
-function samplepair(rng::AbstractRNG, n::Int)
-    i1 = rand(rng, 1:n)
-    i2 = rand(rng, 1:n-1)
+function samplepair(rng::AbstractRNG, n::Integer)
+    i1 = rand(rng, Base.OneTo(n))
+    i2 = rand(rng, Base.OneTo(n - one(n)))
```

Comment:
Isn't this equivalent? I find it more standard and readable:
```suggestion
    i1 = rand(rng, 1:n)
    i2 = rand(rng, 1:(n - one(n)))
```

Reply:
It'd have to be `one(n):(n - one(n))` to avoid promoting smaller types to `Int`. I find that a bit less clear than `Base.OneTo(n - one(n))` but don't feel strongly.